router-tests/ratelimit_test.go

```diff
@@ -669,6 +669,143 @@ func TestRateLimit(t *testing.T) {
 			require.Equal(t, `{"errors":[{"message":"Rate limit exceeded"}],"data":null}`, res.Body)
 		})
 	})
+	t.Run("Cluster Mode", func(t *testing.T) {
+		var (
+			clusterUrls     = "redis://localhost:7000,redis://localhost:7001,redis://localhost:7002"
+			clusterUrlSlice = []string{"localhost:7000", "localhost:7001", "localhost:7002"}
+			password        = "test"
+		)
+
+		t.Run("enabled - below limit", func(t *testing.T) {
+			t.Parallel()
+
+			key := uuid.New().String()
+			t.Cleanup(func() {
+				client := redis.NewClusterClient(&redis.ClusterOptions{Addrs: clusterUrlSlice, Password: password})
+				del := client.Del(context.Background(), key)
+				require.NoError(t, del.Err())
+			})
+			testenv.Run(t, &testenv.Config{
+				RouterOptions: []core.Option{
+					core.WithRateLimitConfig(&config.RateLimitConfiguration{
+						Enabled:  true,
+						Strategy: "simple",
+						SimpleStrategy: config.RateLimitSimpleStrategy{
+							Rate:                    1,
+							Burst:                   1,
+							Period:                  time.Second * 2,
+							RejectExceedingRequests: false,
+						},
+						Storage: config.RedisConfiguration{
+							Url:       clusterUrls,
```

Comment:
According to the router configuration and json schema, only single URL is supported. Could you please provide a full example and update schema description?